src/components/organisms/PageHeader/OPAChip.tsx

```diff
@@ -0,0 +1,15 @@
+import {useAppSelector} from '@redux/hooks';
+
+import {OPAStatus} from '@components/atoms/Icon/Icons';
+
+import * as S from './OPAChip.styled';
+
+export const OPAChip = () => {
+  const opaRulesCount = useAppSelector(state => state.validation.config.rules)?.length || 0;
```

Comment:
The `.length` could go into the selector, it will also help avoiding re-renders if the number doesn't change but the rules themselves change.
```suggestion
  const opaRulesCount = useAppSelector(state => state.validation.config.rules.length || 0);
```